### DISCUSSION TITLE
帮忙看下是Evolved哪个插件、样式、扩展残留了没删干净？

### DISCUSSION THREAD

**Discussion by gsxgmpkt9020** [the1812/Bilibili-Evolved#5082] [综合讨论 / General]
1. “已关注”点一下就能就能取消关注，不需要选择二级菜单
![comic](https://github.com/user-attachments/assets/8b0b4d28-82d4-4ef7-9013-d57a3b22a473)

2. 以关注的UP主以三栏的样式显示
![image](https://github.com/user-attachments/assets/46675441-93d4-4262-aecf-6b49d64fe147)

怎么都排查不出来

**Comment by the1812** [comment]
Evolved 没有这种功能

**Comment by Geeyun-JY3** [comment]
你这是不是新版个人空间？

**Reply by gsxgmpkt9020** [reply]
> 你这是不是新版个人空间？

这个不懂诶。。。有切换小按钮吗？不同浏览器同一个账号都不一样

**Reply by Geeyun-JY3** [reply]
新旧版个人空间样式区别很大啊，你看不出来是新是旧吗？#5011

**Reply by gsxgmpkt9020** [reply]
> 新旧版个人空间样式区别很大啊，你看不出来是新是旧吗？#5011

1. 如果我未卜先知已经知道这是B站官方的所谓“新版空间样式”，我还会在这里问吗？
2. 没有任何证据表明，B站正式大面积官宣过要更改新的空间样式。说不定是临时工跑路前高的恶作剧或者随机bug呢？
3. 请问B站切换新旧样式的按钮再哪里？
4. 如何解释我同一个账号，不同浏览器一个是所谓的官方旧样式，一个是所谓的“官方新样式”？这种随机暗改和木马劫持有什么区别？
5. 没有正式官宣就不能严格证明这是官方行为，所以根本不存在所谓新旧，能否分辨更加无从谈起，只能把它暂时认定为异常现象。我们版区的开源维护者 @the1812 回答的时候也同样没有提起这是所谓的“新版”，所以你口中的所谓新版并不是社区的普遍共识。

**Reply by Geeyun-JY3** [reply]
> 新旧版个人空间样式区别很大啊，你看不出来是新是旧吗？#5011

是我这条回复措辞有问题吗？本意只是指出新旧版个人空间是很容易分辨的，并给出分辨参考，如有误解还望见谅。

> 1. 如果我未卜先知已经知道这是B站官方的所谓“新版空间样式”，我还会在这里问吗？
> 2. 没有任何证据表明，B站正式大面积官宣过要更改新的空间样式。说不定是临时工跑路前高的恶作剧或者随机bug呢？
> 3. 请问B站切换新旧样式的按钮再哪里？
> 4. 如何解释我同一个账号，不同浏览器一个是所谓的官方旧样式，一个是所谓的“官方新样式”？这种随机暗改和木马劫持有什么区别？
> 5. 没有正式官宣就不能严格证明这是官方行为，所以根本不存在所谓新旧，能否分辨更加无从谈起，只能把它暂时认定为异常现象。我们版区的开源维护者 @the1812 回答的时候也同样没有提起这是所谓的“新版”，所以你口中的所谓新版并不是社区的普遍共识。

呃呃呃呃呃我只是猜测这有可能是破站新版主页的特性，因为这个新版个人空间样式和旧版差别较大，所以你那边如果是新版是能直接看出来的，先前就有人报过相关 issue #5011，你可以参考其中截图。
上面提到的 issue #5011 加上我也在 Firefox 上被灰度到过，就足以证明这就是官方正在灰度测试的新版个人空间样式，同一个账号在不同浏览器有不同的灰度情况并不是没可能，破站各种新版页面一直都有搞这种灰度测试，我也不知道破站的灰度具体是怎么开展的。

如果你说的「不同浏览器同一个账号都不一样」是指同一账号在不同浏览器打开的个人空间页面长得不一样，那其中一个浏览器里应该是被灰度到了新版个人空间的。

**Reply by Geeyun-JY3** [reply]
差点忘了我在 #5011 下评论有附截图了。相信能很明显从 https://github.com/the1812/Bilibili-Evolved/issues/5011#issuecomment-2543102573 附的截图看到破站这个个人空间切换新旧样式的按钮就在右下角。
补充一句，因我被灰度到时未登录测试过，故有上述猜测。
